### DISCUSSION TITLE
Bug: Not enough information in stack trace

### DISCUSSION THREAD

**Discussion by Nefcanto** [vercel/next.js#27350] [Help]
### What version of Next.js are you using?

11.0.1

### What version of Node.js are you using?

node:lts-alpine

### What browser are you using?

Chrome 91.0.4472.164 (Official Build) (64-bit)

### What operating system are you using?

Ubuntu 20.04 (and docker node:lts-alpine)

### How are you deploying your application?

Dockerfile, then inside docker `next build`

### Describe the Bug

When I want to build my website, I see this error:

```
info  - Generating static pages (3/5)

Error occurred prerendering page "/". Read more: https://nextjs.org/docs/messages/prerender-error
Error: Minified React error #143; visit https://reactjs.org/docs/error-decoder.html?invariant=143 for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at Object.only (/Company/Site/node_modules/react/cjs/react.production.min.js:19:241)
    at Link (/Company/Site/.next/server/chunks/61.js:685:29)
    at d (/Company/Site/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:33:498)
    at bb (/Company/Site/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:36:16)
    at a.b.render (/Company/Site/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:42:43)
    at a.b.read (/Company/Site/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:41:83)
    at exports.renderToString (/Company/Site/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:52:138)
    at Object.renderPage (/Company/Site/node_modules/next/dist/next-server/server/render.js:53:854)
    at Function.getInitialProps (/Company/Site/.next/server/pages/_document.js:508:19)
    at loadGetInitialProps (/Company/Site/node_modules/next/dist/next-server/lib/utils.js:5:101)
```

This has no reference to **my code**. I can't understand what the problem is.

A typical website can have pages that can have at least 500 lines of code. This stack trace does nothing in troubleshooting the error.

And when I go to the URL you have provided, all I see is this:

> React.Children.only expected to receive a single React element child.

It's a little help. But where should I find this? What file? What line? Of course it's not possible to scan an entire 10K codebase to guess what the problem might be.

React version: 17.0.2

## Steps To Reproduce

I don't know.

Please don't close this issue. I have a complete website and I don't know what should be done to recreate the issue.

## The current behavior

Next Build engine does not give us information about **our code**. This makes debugging extremely hard if not impossible.

## The expected behavior

Tell us what file and what line has the problem.

### Expected Behavior

It should give us the address of the problem:

1. What file?
2. Which line?

### To Reproduce

I don't know how to reproduce this error.

**Comment by mattcarlotta** [comment]
Without a [mcve](https://stackoverflow.com/help/minimal-reproducible-example), my **guess** is that it's related to how you're rendering children within a [Next Link](https://github.com/vercel/next.js/blob/canary/packages/next/client/link.tsx#L231-L246).

If you can provide _temporary_ access to your codebase, it'll be easier to debug.

**Comment by Nefcanto** [comment]
@mattcarlotta, I will gladly give you access. I also would gladly give you remote access to my laptop. Because we're building inside docker, and we have an SPL (software product line) that needs configuration to run. But I would gladly proceed as you tell.

**Comment by mattcarlotta** [comment]
Some questions before we proceed:
1.) Are you able to build this application locally?
2.) Does building locally display the same error?
3.) Is the error consistent? Or does it only happen some of the time?

**Comment by Nefcanto** [comment]
OK, let me test those questions first, and also test your `Link` issue you mentioned. I'll come back to you with answers. Thanks.

**Comment by ijjk** [comment]
Hi, do note that you may be able to get a better error visiting the mentioned page that failed to prerender while running the application in dev mode (`next dev`)
